tests/lcb/stdlib/arithmetic.lcb

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright (C) 2015 Runtime Revolution Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+module com.livecode.arithmetic.tests
+
+public handler TestUnarySigns()
+	variable t
+	put 10 into t
+	test "unary−" when -t is -10
+
+	put -10 into t
+	test "unary+" when +t is -10
+end handler
+
+public handler TestAdd()
+	variable tInt as integer
+	put 10 into tInt
+
+	add 2 to tInt
+	test "add (syntax, int)" when tInt = 12
+	test "add (operator, int)" when tInt + 1 = 13
+
+	add 1.9 to tInt
```

Comment:
This will actually throw an error with my integer changes. 'add real to int' should use 'add number to number', but the out value won't match integer, so you'll get a typecheck error.